LayoutKitSampleApp/Benchmarks/DataBinder.swift

```diff
@@ -6,9 +6,44 @@
 // software distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 
-import Foundation
+import UIKit
 
 protocol DataBinder {
-    associatedtype DataType
-    func setData(_ data: DataType)
+    func setData(_ data: FeedItemData)
+}
+
+enum DataBinderHelper {
+
+    /// Before v4.0.2, `CollectionViewController` and `TableViewController` are using generics.
+    /// `DWURecycling` will use method swizzling to replace some `UIcollectionView` and `UITableView`
+    /// data source methods and inject label to the cells.
+    /// By using different ContentView, `TableViewController` and `CollectionViewController` will be
+    /// treated differently. That means the correct method swizzling is only done correctly for the for
+    /// time. The solution to remove generic is a fix to https://github.com/linkedin/LayoutKit/issues/103
```

Comment:
This comment is unclear to me.
What does "will be treated differently" mean here specifically?
Typo in "for the for time".
It says that "`DWURecycling` will use method swizzling" -- isn't it using it currently?
It'd be good to explain the problem in more detail.
Also, a comment explaining the new approach would help too.